packages/@webex/internal-plugin-metrics/src/call-diagnostic/config.ts

```diff
@@ -388,6 +395,18 @@ export const CLIENT_ERROR_CODE_TO_ERROR_PAYLOAD: Record<number, Partial<ClientEv
     category: 'media',
     fatal: true,
   },
+  2050: {
```

Comment:
nitpick: should we be using some constants for these?

Reply:
for the code itself? most of the codes don't in this map, but i am happy to if we'd like. it seemed split so i wasn't sure what pattern to follow.

Reply:
the code is used in more than 1 place so kind of makes sense to have a const, but on the other hand these numbers are pretty unique, so I don't mind that much - that's why I said it's a "nitpick",